wandb/cli/cli.py

```diff
@@ -1680,6 +1680,9 @@ def launch(
     hidden=True,
     help="a wandb client registration URL, this is generated in the UI",
 )
+@click.option(
+    "--verbose", "-v", count=True, default=False, help="Display verbose output"
```

Comment:
i think you have a typo: i think `default=False`, should be the `default=0`, but in this case default is 0, so you can just omit it